free-programming-books-de.md

```diff
@@ -115,6 +116,7 @@
 * [Java 7 Mehr als eine Insel](http://openbook.rheinwerk-verlag.de/java7/)
 * [Java ist auch eine Insel](http://openbook.rheinwerk-verlag.de/javainsel/)
 * [Java SE 8 Standard-Bibliothek](http://openbook.rheinwerk-verlag.de/java8/) - Christian Ullenboom
+* [Nebenläufige Programmierung mit Java (2017)](https://www.assets.dpunkt.de/openbooks/Hettel_Nebenlaeufige%20Programmierung%20mit%20Java_Broschuere.pdf) - Jörg Hettel, Manh Tien Tran (PDF)
```

Comment:
this is an advertising sample, not a book

Reply:
you are right, sorry for the mistake